recipes/pbzip2/build.sh

```diff
@@ -0,0 +1,2 @@
+make PREFIX=$PREFIX LDFLAGS="-L$PREFIX/lib" CXXFLAGS="-I$PREFIX/include $CXXFLAGS"
```

Comment:
Again, just a note but you are overwriting `LDFLAGS` here as you didn't include the existing variable (as you did in `CXXFLAGS`).